### ISSUE TITLE
Unable to call Argo scripts with Quest query object

### ISSUE THREAD

**Issue by zachghiaccio** [icesat2py/icepyx#453]
This issue was found while working on the **argo** branch.

I'm working on an example notebook to showcase the QUEST functionality, but I am having a hard time accessing the scripts needed to query Argo data. I am able to access the scripts and do an Argo search with the following import statements:
```
import icepyx as ipx
from icepyx.core.spatial import geodataframe
from icepyx.quest.dataset_scripts.dataset import DataSet
from icepyx.quest.dataset_scripts.argo import Argo
```

However, when attempting to access Argo (and ICESat-2) data through the following import:

```
from icepyx import Quest
```

I am having a tough time finding a querying function for Argo. I see there is a `Quest().add_icesat2()` feature, and there is a similar function for Argo in `test_quest.py` (Line 35), but it does not seem to be implemented.

**Comment by JessicaS11** [comment]
Hey @zachghiaccio! We implemented this functionality in #441 but commented it out so that the tests would pass (because the `argo.py` script the functions in `quest.py` called wasn't actually available yet in that PR). I've uncommented the `Quest.add_argo()` pieces (and related tests) in the argo branch (#427). Let me know if that doesn't resolve this (note that kwargs won't be accepted until we merge #452).
